### PR TITLE
Convert registerShapeMethod cases to registerMethod

### PR DESCRIPTION
**Pull request by pentacular**
This allows constructors to take context.

e.g., shape.Box(diameter()) will construct a box with the diameter of shape.

### CODE REVIEW

File: api/shape/Arc.js [hunk 1]
```diff
@@ -165,9 +171,9 @@ const ArcOp =
     return result;
   };
 
-export const Arc = Shape.registerShapeMethod('Arc', ArcOp('Arc'));
-export const ArcX = Shape.registerShapeMethod('ArcX', ArcOp('ArcX'));
-export const ArcY = Shape.registerShapeMethod('ArcY', ArcOp('ArcY'));
-export const ArcZ = Shape.registerShapeMethod('ArcZ', ArcOp('ArcZ'));
+export const Arc = Shape.registerMethod('Arc', ArcOp('Arc'));
```
Comment (BarbourSmith) [inline api/shape/Arc.js:174]: Nice. Shape.registerShapeMethod seems redundant 😛

File: geometry/tagged/computeToolpath.js [hunk 2]
```diff
@@ -51,10 +51,12 @@ export const computeToolpath = (
     stopCost = 30,
     candidateLimit = 1,
     subCandidateLimit = 1,
+    radialCutDepth = toolDiameter / 4,
```
Comment (BarbourSmith) [inline geometry/tagged/computeToolpath.js:54]: 👍 This is a good default...1/3 would probably be safe too

**Review by BarbourSmith — APPROVED** [review]
👍 👍 

There are a couple visual diff changes that I'm not 100% sure about...not that they are wrong, just that I wasn't sure. Like this sort of thing:

image

**Comment by pentacular** [comment]
> +1 +1
> 
> There are a couple visual diff changes that I'm not 100% sure about...not that they are wrong, just that I wasn't sure. Like this sort of thing:
> 
> image

Yeah, I think this one is correct now -- it should produce a group with both parts of the cut.

Not sure how it got to the earlier state -- need to be a bit more careful. :)

File: api/shape/Arc.js (hunk 1 above)
Comment (pentacular) [reply api/shape/Arc.js:174]: Yeah, it was necessary before the new chaining system was in effect, but now we can remove it as a special case. :)

**Pull request merged**